fiatconnect-api.md

```diff
@@ -1796,6 +1796,897 @@ is below:
 
 * `'NG'`: The account number should be exactly 10 digits long, and only include the numbers 0-9.
 
+####  7.3.2.2. `CheckingAccountIfsc`
+
+`CheckingAccountIfsc` is a Fiat Account schema that represents accounts for transactions across `INR` currency.
+
+```
+{
+    "accountName": `string`,
+    "institutionName": `string`,
+    "ifsc": `string`,
+    "accountNumber": `string`,
+    "country": `string`,
+    "fiatAccountType": "CheckingAccountIfsc"
+}
+```
+####  7.3.2.3. `CheckingAccountIban`
+
+`CheckingAccountIban` is a Fiat Account schema that represents accounts for transactions across `EUR` currency.
+
+
+```
+{
+    "accountName": `string`,
+    "institutionName": `string`,
+    "iban": `string`,
+    "accountNumber": `string`,
+    "country": `string`,
+    "fiatAccountType": "CheckingAccountIban"
+}
+```
+
+####  7.3.2.4. `CheckingAccountBankCode`
+
+`CheckingAccountBankCode` is a Fiat Account schema that represents accounts for transactions across `NG`, `GH`, `KE`, `ZA`, `PH`, `SN`, `CN`, `GB`, `SG`, `TZ`, `HK` currencies.
+
+
+```
+{
+    "accountName": `string`,
+    "institutionName": `string`,
+    "bankCode": `string`,
+    "accountNumber": `string`,
+    "country": `string`,
+    "fiatAccountType": "CheckingAccountBankCode"
+}
+```
+
+####  7.3.2.5. `CheckingAccountMobile`
+
+`CheckingAccountMobile` is a Fiat Account schema that represents accounts for transactions across `NG`, `GH`, `KE`, `UG`, `SN`, `BJ`, `ML`, `CI`, `NE`, `TG`, `GW`, `BF`, `SL`, `RW`, `CM`,`CG` currencies.
```

Comment:
There is already a mobile money fiat account schema, I'd prefer if we didn't tack on another one that represents pretty much the same thing. I suggest you add the mobile money providers you want to support-- MTN, TIGO, and Vodafone-- to the SupportedOperatorEnum rather than create a new bespoke FiatAccount schema that represents the same kind of account

Reply:
We had initially done it in the proposed method of allowdValues but on the partner call, it was suggested that we separate the schema for each.

Reply:
@cajubelt - removed the mobile schema, we will utilise the other schema which is already for mobile money